build/cpp/warble/src/warble_functions.cc

```diff
@@ -81,34 +55,24 @@ warble_functions::WarbleFunctions::WarbleRequestPackager(const Any &any) {
   // if parent id is empty, it is a base warble
   // otherwise, add the warble to the parent id
   if (warble.parent_id() != "") {
-    if (std::stoi(warble.parent_id()) >= warble_count_) {
-      LOG(ERROR) << "WarbleRequest packaged with invalid parent_id";
-      // -1 indicates an error has occurred
-      const auto &warble_put = std::make_tuple(
-          -1, "warble_" + std::to_string(warble_count_), serialized_warble);
-      puts.push_back(warble_put);
-    } else {
-      LOG(INFO) << "WarbleRequest packaged with non-empty parent_id";
-      const auto &warble_put = std::make_tuple(
-          0, "warble_" + std::to_string(warble_count_), serialized_warble);
-      const auto &parent_put =
-          std::make_tuple(0, "warble_" + warble.parent_id(), serialized_warble);
-      puts.push_back(warble_put);
-      puts.push_back(parent_put);
-      warble_count_++;
-    }
+    LOG(INFO) << "WarbleRequest packaged with non-empty parent_id";
+    const auto &warble_put =
+        std::make_tuple(0, "warble_" + warble.id(), serialized_warble);
+    const auto &parent_put =
+        std::make_tuple(0, "warble_" + warble.parent_id(), serialized_warble);
+    puts.push_back(warble_put);
+    puts.push_back(parent_put);
```

Comment:
No longer checking for valid parent ids as this is carried out at the command line.